### DISCUSSION TITLE
Instead of client.send() use the client.json()

### DISCUSSION THREAD

**Discussion by aravindbalaji-ngp** [adonisjs/core#4097] [Cookbooks]
While testing adonisjs i came across this problem where i need to post data via the tester but in the documentation there were no direct answers. But if you search through the functions available for the client object, you will come across a method called .json() which is supported in adonisjs v5, because in adonisjs v5 client.send() method no longer accepts any data in its params.

Use this: 
`const response = await client.post('/employees').json(data)`

Instead of this:
`const response = await client.post('/employees').send(data)`

**Comment by RomainLanz** [comment]
Hey @aravindbalaji-ngp! 👋🏻 

I am unsure where you saw the syntax using `send()`.
AdonisJS uses [`Japa`](https://japa.dev/docs) to run your test. All methods are properly documented in their documentation.

i.e: https://japa.dev/docs/plugins/api-client#json

**Reply by aravindbalaji-ngp** [reply]
Hi @RomainLanz 

It was in your adonisjs 4.1 docs - https://legacy.adonisjs.com/docs/4.1/api-tests#_client_methods
